scripts/es-integration-test.sh

```diff
@@ -96,14 +96,32 @@ teardown_storage() {
   docker compose -f "${compose_file}" down
 }
 
+build_local_img(){
+    make "build-binaries-linux"
+    make create-baseimg
+    #build es-index-cleaner
+    docker build \
```

Comment:
This worked for me:

Build with a fake `GITHUB_SHA/BRANCH=local-test` variables
```
GITHUB_SHA=local-test BRANCH=local-test bash scripts/build-upload-a-docker-image.sh -l -b -c jaeger-es-index-cleaner -d cmd/es-index-cleaner -t release -p linux/$(go env GOARCH)
```
Run as `local-test` from **local registry**
```
docker run localhost:5000/jaegertracing/jaeger-es-index-cleaner:local-test
```